docs/docs/getting-started/install-with-docker.md

```
@@ -77,8 +76,8 @@ docker compose up -d
 This will start both the application and the database services in detached mode.
 
 You should then be able to connect to the following services:
-- the API at http://localhost:8000/docs
-- the webapp at http://localhost:8501
+- the API at http://localhost:8000/api-docs
+- the Docusaurus platform at http://localhost:3000
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)